libs/services/designer-client-services/src/lib/base/operationmanifest.ts

```diff
@@ -112,7 +112,7 @@ const xslttransform = 'xslttransform';
 const datamapper = 'datamapper';
 
 export const apiManagementConnectorId = '/connectionProviders/apiManagementOperation';
-export const azureFunctionConnectorId = '/connectionProviders/azureFunctionOperation';
```

Comment:
I am not sure if backend is using this connectorId, please double check on what standard service is returning